packages/admin/cms-admin/src/common/image/ImageCrop.sc.ts

```diff
@@ -78,12 +87,16 @@ export const FocalPointHandle = styled("button")<FocalPointHandleProps>`
         border-radius: 50%;
         background-color: ${({ theme }) => theme.palette.primary.main};
         margin: 9px;
+        /* transform: translate(-50%, -50%); */
+        /* left: 50%; */
+        /* top: 50%; */
+        transition: all 0.5s ease;
```

Comment:
What do you want to animate? Please specify the respective property instead of "all"